cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -672,24 +656,20 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size)
     int next_valid_count;
 
     // only need to process definition levels if this is a nullable column
```

Comment:
```suggestion
    // only need to process definition levels if the column has nulls
```

Reply:
I think the same comment was at another place as well that may also be updated 😊